src/bundle/Command/MigrateNamespacesCommand.php

```diff
@@ -0,0 +1,206 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Bundle\FieldTypeRichText\Command;
+
+use Ibexa\Contracts\Core\Repository\PermissionResolver;
+use Ibexa\Contracts\Core\Repository\UserService;
+use Ibexa\FieldTypeRichText\Persistence\Legacy\ContentModelGateway as Gateway;
+use Symfony\Component\Console\Exception\RuntimeException;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+final class MigrateNamespacesCommand extends AbstractMultiProcessComand
```

Comment:
Naming: It's very ambiguous. Rather: `MigrateXMLNamespacesCommand`. I'm not including here RichText since we're already in the RichText PHP namespace.